for-upstream/common.nix

```diff
@@ -104,23 +116,16 @@ in
         message = "config.nix.package must match the version of Nix used in hercules-ci-agent. If you are trying to use a different version of Nix, try overriding the pkgs.nix attribute by means of an overlay.";
       }
     ];
-    systemd.services.hercules-ci-agent = {
-      wantedBy = [ "multi-user.target" ];
-      after = [ "network.target" ];
-      serviceConfig = {
-        User = cfg.user;
-        ExecStart = "${cfg.package}/bin/hercules-ci-agent --config ${tomlFile}";
-        Restart = "on-failure";
-        RestartSec = 120;
-        StartLimitBurst = 30 * 1000000; # practically infitine
-      };
-    };
+    services.hercules-ci-agent = {
+      inherit finalConfig tomlFile;
+
+      # TODO: expose only the (future) main directory as an option and derive 
+      # all locations from finalConfig.
+      clusterJoinTokenPath = lib.mkDefault "/var/lib/keys/hercules-ci-agent/cluster-join-token.key";
```

Comment:
This will violate FHS and such, it's going to be harder to get upstream than `/var/lib/hercules-ci-agent/keys`

Reply:
It's copied from the old nixops profile. This default will change once we unify the directories anyway, so we can leave it for now.